style.css

```diff
@@ -0,0 +1,122 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  position: absolute;
+  top: 1px;
+  background-image: url('font.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.top-navigation {
+  display: block;
+}
+
+.nav-bar {
+  display: block;
+  flex-direction: row;
+  padding: 26px 80px;
+  width: 100%;
+  height: 72px;
+  z-index: 1;
+  box-shadow: 0 1px 10px -5px grey;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
```

Comment:
Nice job done so far on your layouts👍 But we are required to use flexbox to layout contents in this milestone🤗🤗🤗